ui/modal_templates.go

```diff
@@ -146,6 +149,18 @@ func (m *ModalTemplate) changePassword() []func(gtx C) D {
 	}
 }
 
+func (m *ModalTemplate) verifyMessageInfo() []func(gtx C) D {
+	return []func(gtx C) D{
+		func(gtx C) D {
+			text := m.th.Body1("After you or your counterparty has genrated a signature, you can use this form to verify the" +
+				" validity of the  signature. \n \nOnce you have entered the address, the message and the corresponding " +
+				"signature, you will see VALID if the signature appropriately matches the address and message, otherwise INVALID.")
+			text.Color = m.th.Color.Gray
+			return text.Layout(gtx)
```

Comment:
```suggestion
func (m *ModalTemplate) verifyMessageInfo(infoText) []func(gtx C) D {
	return []func(gtx C) D{
		func(gtx C) D {
			text := m.th.Body1(infoText)
			text.Color = m.th.Color.Gray
			return text.Layout(gtx)
```

Rather than we declare multiple info modals, we can have one model that takes the info text as a parameter and can be used by all subpages with the info option.